tests/utils/kudo/kudo.go

```diff
@@ -21,8 +24,11 @@ import (
 )
 
 var (
-	kubectlOptions *kubectl.KubectlOptions
-	kudo           *versioned.Clientset
+	kubectlOptions       *kubectl.KubectlOptions
+	kudo                 *versioned.Clientset
+	operatorYamlFilePath = "../../operator/operator.yaml"
+	// https://regex101.com/r/Ly7O1x/3/
+	semVerRegexp = `(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?`
```

Comment:
Better: https://github.com/Masterminds/semver

Reply:
Oh, you don't want to check if a version is semver but instead extract the operator version from `operator.yaml`. Probably don't need this regex, you could simply assume that the string following `operatorVersion:` is semver because you can control this in this test env.